### PR TITLE
chore for setting up houndCI

### PR DESCRIPTION
**Pull request by patrickhag**
[#187354233](https://www.pivotaltracker.com/story/show/187354233) 

## Purpose

Describe the purpose of this pull request

- The purpose of this PR is to give each and every PR that's created to be linted by HoundCI using ESLint

## Changes Made

* Created `.hound.yml` file
* Inserted a badge in `README.md` using [shields](https://shields.io/)

## Testing Instructions

None

## Related Issues

None

## Checklist

Please review the following checklist and make sure all tasks are complete before submitting:

- [x] Code follows the project's coding standards
- [x] Changes are covered by tests
- [x] Documentation is updated (if applicable)
- [x] All tests pass

### CODE REVIEW

**Review by Mag-codes — APPROVED** [review]

**Review by Favor-star — APPROVED** [review]
Yeah It's fine!

File: README.md
Comment (hozayves) [inline README.md:1]: Hi @patrickhag ,

Could you please provide some insight into the changes made in this file? I'm curious to understand the reasons behind the modifications. Thanks!

Comment (patrickhag) [reply README.md:1]: Yes mate, What I did is that I just added a badge indicating that our project is being reviewed by HoundCI. And I did that using [shields](https://shields.io/)

Shields typically refers to small visual indicators that are displayed in a project's README file to provide quick information about the project's status, such as build status, code tests coverage, version, etc.

**Review by JeanIrad — APPROVED** [review]
great work

**Review by niyontwali — APPROVED** [review]
Well done

**Pull request merged**